src/main/java/net/svera/RestAPISvera/expenses/models/Expense.java

```diff
@@ -0,0 +1,29 @@
+package net.svera.RestAPISvera.expenses.models;
+
+import lombok.Data;
+import lombok.NonNull;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@Table(name="EXPENSE")
+public class Expense {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String location;
+    @NonNull private String name;
+    @NonNull private Date date;
+    @NonNull  private double price;
+    private double quantity;
+    private String unit;
+    private Date expireDate;
+
+    @ManyToOne
+    @JoinColumn(name="subcategory_id", nullable = false)
```

Comment:
Would it be better to have categories and subcategories in one table, as multi level? Here I have to have subcategory, while main category can't be used. What if there would be 3 levels?